robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -46,6 +46,25 @@ def click_gift_entry_button(self,title):
         self.selenium.scroll_element_into_view(locator)
         self.selenium.click_element(locator)
 
+    def add_batch_table_columns(self,*args):
```

Comment:
we already have this right? why are u adding this keyword again?

Reply:
yeah this is an accident. Thanks for pointing out.